funds/privacy_guides/privacy_guides_funds.md

```diff
@@ -0,0 +1,163 @@
+* The committee votes unanimously to close the Fund. MAGIC Grants may optionally also require a Voter majority vote.
+* The committee votes by majority vote to “split” the Fund into several Funds. This is intended for major Project changes or splits (eg: Bitcoin/Bitcoin Cash). MAGIC Grants may optionally “freeze” the existing Fund while it is determining how to allocate assets among the two new Funds. MAGIC Grants may deny the formation of new Funds, preventing the “split” in effect. Should MAGIC Grants agree to “split” the Fund, Voters must agree by majority vote to 1) close the existing Fund, 2) open two (or more, as applicable) new Funds under new charter documents, and 3) vote to approve the allocation of assets among the new Funds. The new Funds will then undergo the typical Fund startup processes of selecting Voters and electing a committee, subject requirements listed in those new Fund charter documents.
+
+## Renaming the Fund
+
+The committee can vote by majority vote to rename the Fund, which must be approved by MAGIC Grants.
+
+## Changes to this Document
+
+Unless other restrictions apply to the relevant sections, the committee may modify this document by majority vote. MAGIC Grants must approve the changes, and it may require a Voter majority vote to approve the changes even if this is not explicitly required by the existing language. Unless waived by MAGIC Grants, proposed changes should be announced publicly at least 30 days in advance of the change.
+
+Should any portion of this document affect the legal standing of MAGIC Grants, MAGIC Grants may change the relevant portions without any other approvals and without notice.
+
+## MAGIC Grants Existing Board
+
+[Link](/about/)
+
+## Contact Information
+
+MAGIC Grants: [info@magicgrants.org](mailto:info@magicgrants.org)
+
+MAGIC **Project** Fund (emails all committee members): [<>@magicgrants.org](mailto:<>@magicgrants.org)
+
+## Appendix A - Restrictions on Grant Disbursements
+
+In addition to all the normal restrictions on grant disbursements, this MAGIC Fund shall have the following additional restrictions on grant disbursements:
+
+* None
+
+## Appendix B - MAGIC Fund Voter Modifications
+
+Modifications to MAGIC Fund Voters:
+
+* There are no MAGIC Fund Voters for this MAGIC Fund.
+
+## Appendix C - MAGIC Fund Committee Members and Elections
+
+Initially, there will be 5 committee members. None of the committee members are elected, and they will serve indefinite terms.
+
+## Appendix D - Committee Allowable Compensation and Restrictions
+
+The amount allowable for committee member compensation is the lesser value of the restrictions imposed by Parts A and B:
+
+Part A: This is the compensation allowable to committee members by Voters (modifiable by board-approved Voter majority vote):
+
+* Term: Monthly
+* Amount: $0
+* Maximum amount per month: $0
+* Maximum amount per year: $0
```

Comment:
Does this refer exclusively to compensation received solely for being a committee member, or would it extend to reimbursement for expenses or payment for actual time worked as well?

Reply:
It does not apply for expenses, but in general, it would cover time worked as well. If you want the committee to be able to be compensated for their time, let me know.

Being compensated for their work *as a committee member* instead of *as a contracted person by the committee* is strongly preferred.